# This patch fix the error generated by newer version of perl (5.26, or
# above) when it handled '{' literally (by excalping that).
# This bug is fixed upstream, and update for pkgbuild should address this. 

--- pkgbuild-1.3.105/config.pm.orig	2008-08-10 16:43:57.000000000 +0000
+++ pkgbuild-1.3.105/config.pm	2019-06-20 03:08:54.379800455 +0000
@@ -1,7 +1,7 @@
 #
 #  The pkgbuild build engine
 #
-#  Copyright (C) 2004, 2005 Sun Microsystems, Inc.
+# Copyright (c) 2004, 2019, Oracle and/or its affiliates. All rights reserved.
 #
 #  pkgbuild is free software; you can redistribute it and/or
 #  modify it under the terms of the GNU General Public License 
@@ -225,7 +225,7 @@
 	return undef;
     }
 
-    while ($str =~ /%{([a-zA-Z]+)}/) {
+    while ($str =~ /%\{([a-zA-Z]+)}/) {
 	my $var = $1;
 	if (defined ($self->{valid_keys}->{$var})) {
 	    my $val = $self->get ($var);
